Makefile

```diff
@@ -58,7 +63,20 @@ CALYPSO_ENV ?= $(NODE_ENV)
 
 export NODE_ENV := $(NODE_ENV)
 export CALYPSO_ENV := $(CALYPSO_ENV)
-export NODE_PATH := server:client:.
+export NODE_PATH := server$(SEPARATOR)client$(SEPARATOR).
+
+# We use `semver` to check the version of Node.js before installing npm
+# packages or running scripts.  Since this is before npm runs, we need to grab
+# the version of `semver` installed with the npm executable.
+ifeq ($(OS),Windows_NT)
+# On Windows, the npm global install path is under AppData:
+# http://stackoverflow.com/a/26894197
+# `semver` is in the base `node_modules` folder for the `node` installation.
+export SEMVER_GLOBAL_PATH := $(dir $(shell which $(NODE)))/node_modules/npm/node_modules/semver
```

Comment:
Good call on the variable name (and adding `semver` to it) - now it's more precise.